prdoc/pr_4764.prdoc

```diff
@@ -0,0 +1,19 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Secretary Program
+
+doc:
+  - audience: Runtime User
+    description: |
+      The Secretary Collective is an on-chain body comprising a single rank, Secretary, with the function to oversee the activities of the Polkadot Technical Fellowship. 
+      The Secretary is responsible for ensuring that the Fellowship's activities are carried out efficiently and in accordance with established protocols.
+      This role includes monitoring, reporting, and providing guidance to the Fellowship to help achieve its goals within the Polkadot ecosystem.
```

Comment:
This sounds like it is made for Polkadot, but it's for test network

Reply:
Thank you for the review!

It should not reflect Polkadot?